app/frontend/src/components/designer/FormSettings.vue

```diff
@@ -245,12 +259,13 @@
                       v-on="on"
                     />
                   </template>
-                  <span
+                  <span :lang="lang"
                     >{{ $t('trans.formSettings.experimental') }}
                     <a
                       :href="githubLinkScheduleAndReminderFeature"
                       class="preview_info_link_field_white"
                       :target="'_blank'"
+                      :hreflang="lang"
```

Comment:
An `href` snuck in here (probably from VSCode autocomplete).

Reply:
It is not a vs-code autocomplete. It is an attribute Jordan mentioned that we should add to link for multilanguage support